test/new-e2e/tests/agent-metric-logs/log-agent/listener/testdata/udp-compose.yaml

```diff
@@ -0,0 +1,23 @@
+---
+services:
+
+  logger-app:
+    environment:
+      TCP: true
+      TARGET: "agent:10518"
+    labels:
+      com.datadoghq.ad.logs: |
+        [
+          {
+            "type": "udp",
+            "port": 10518,
+            "source": "logger",
+            "service": "test-app"
+          }
+        ]
+
+  agent:
+    environment:
+      DD_LOGS_ENABLED: true
+      DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL: false
```

Comment:
FYI `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` is false by default. (unless this is overridden to true by default somewhere in the test infra) - so you may be able to omit this setting here if you want.